src/Console/Command/ProcessCommand.php

```diff
@@ -179,9 +188,10 @@ private function processFile(SplFileInfo $fileInfo, int &$i): void
             $newContent = $this->fileProcessor->processFileToString($fileInfo);
 
             if ($newContent !== $oldContent) {
-                $this->consoleStyle->writeln(sprintf('<options=bold>%d) %s</>', $i, $fileInfo->getPathname()));
-                $this->consoleStyle->writeln($this->differAndFormatter->diffAndFormat($oldContent, $newContent));
-                ++$i;
+                $this->diffFiles[] = [
+                    'file' => sprintf('<options=bold>%d) %s</>', ++$i, $fileInfo->getRelativePathname()),
```

Comment:
`++$i` used in the code take some thinking what the value is.
Could you seperate it before/after this line and use `$i` only?

Reply:
I initialize it as `0` fell lines above. Should I revert as well?

Reply:
On second though, this look quite ok. I checked on mobile and it was too many changes.

So :+1:  for the way it is now!